views/service.html

```diff
@@ -9,12 +9,12 @@
     <dd>{{resource.metadata.creationTimestamp}}</dd>
     <dt>IP</dt>
     <dd>{{resource.spec.portalIP}}</dd>
-    <dt>Port</dt>
-    <dd>{{resource.spec.port}}</dd>    
-    <dt>Container port</dt>
-    <dd>{{resource.spec.containerPort}}</dd>
-    <dt>Protocol</dt>
-    <dd>{{resource.spec.protocol}}</dd>
+    <dt>Port(s)</dt>
```

Comment:
we can probably always say "Ports"... we're presenting them as a list, so that doesn't read too oddly